packages/cloud_firestore/cloud_firestore/macos/cloud_firestore.podspec

```diff
@@ -56,6 +56,7 @@ Pod::Spec.new do |s|
   s.dependency 'firebase_core'
   s.dependency 'Firebase/CoreOnly', "~> #{firebase_sdk_version}"
   s.dependency 'Firebase/Firestore', "~> #{firebase_sdk_version}"
+  s.dependency 'nanopb'
```

Comment:
```suggestion
  # required until firestore-ios-sdk-frameworks is updated, otherwise users of that distribution will have compile failures
  # see https://github.com/invertase/firestore-ios-sdk-frameworks/issues/59
  s.dependency 'nanopb', '>= 2.30908.0', '< 2.30910.0'
```